### PR TITLE
Added InforSuite to 9.1 Compatible product list

### LINKED ISSUE #1248: Please add InforSuite Application Server V11 as Compatible product of Jakarta EE 9.1 Full Platform
To include your Compatible Software Product on the Jakarta EE official list of Compatible Software Products, please provide the following information:

**The URL to the Eclipse Foundation profile to confirm that you are an authorized representative of the company whose product information you are submitting**
https://www.eclipse.org/membership/showMember.php?member_id=1487
**A product name and version**
InforSuite Application Server V11

**The Profile (Full Platform, Web Profile, Both) that the Compatible Product is submitted to be listed under**

**A ZIP file with a product logo with a transparent background in SVG format (optional)**
https://github.com/codff/InforSuite/archive/main.zip

**The Participant, Enterprise or Strategic Member name supporting the Compatible Software Product**
Shandong Cvicse Middleware Co.,Ltd.

**The URL to the approved certification issue demonstrating compatibility as defined in the Jakarta EE TCK Process**
Full Platform:https://github.com/eclipse-ee4j/jakartaee-platform/issues/397
**The URL where Compatible Software Product is available for download**
http://www.inforbus.com/as.html

### PR DESCRIPTION
**Pull request by linkfang**
Fixes [issue1248](https://github.com/jakartaee/jakarta.ee/issues/1248)

### CODE REVIEW

**Review by autumnfound — APPROVED** [review]
LGTM, waiting on approval of TCK results before merging

**Review by ivargrimstad — APPROVED** [review]
The CCR has been approved, so please go ahead!

**Pull request merged**
